src/scene/materials/standard-material.js

```diff
@@ -1129,7 +1136,8 @@ function _defineMaterialProps() {
     _defineFloat('metalness', 1);
     _defineFloat('anisotropy', 0);
     _defineFloat('clearCoat', 0);
-    _defineFloat('clearCoatGlossiness', 1);
+    _defineFloat('clearCoatGloss', 1);
+    _defineFlag('clearCoatGlossInvert', false);
```

Comment:
I think the flag should be defined with the other flags. We tend to keep this separated on define type, not on it's use.

Reply:
I've moved that.

Though it would probably make more sense to order these definitions similar to the class jsdoc order (or material inspector order) - i.e. _logically_ instead of by data type. Doing so should make visually comparing the lists of data between engine/schema/parameters more natural. To me, it makes sense for all parameters affecting the diffuse component to be grouped together, then specular, gloss etc.

Reply:
Perhaps alphabetical order would make most sense...

Reply:
I agree, but then we should change them all in my opinion 🙂

Reply:
> we should change them all in my opinion
yes!

The more I think about it, the more it makes sense we reorder material members to strict alphabetical. Then we can reorder everything else too and comparing diffs becomes much more straight-forward.

I won't do that that here though, of course.